### ISSUE TITLE
[feature request] Better navigation of directories

### ISSUE THREAD

**Issue by rieje** [easymodo/qimgv#252]
Better navigation of directories, like search-as-you-type (e.g. Emac's find-file). Right now, traversing through the file hierarchy requires the mouse and that's the only time I ever need to use a mouse on qimgv. Probably low priority and seems like it will  take some work.

Another tangentially-related feature request is to allow bookmarks the option to include all files in sub-directories. For example, I have a folder named "wallpapers" containing two folders: "portrait" and "landscape". I would like to see all vertical and horizontal wallpapers together, i.e. all images in "wallpapers" recursively. I think this is useful in general because it allows similar images to be displayed together in such a way that it still respects the underlying file hierarchy (i.e. you're not required to put all the photos together directly under one folder in order to see them all in qimgv).

**Comment by markesmith** [comment]
The recursive directory request is a very good call indeed. I too have a large reference images folder that I have divided into sub categories using sub folders. It would be great to select a top level folder and see all images displayed within it including sub directories.

**Comment by LnLcFlx** [comment]
I would love the ability to recursively show images from subdirs!
Some applications save images in a fashion where you have nexted dirs from 0 to f and its a pain to go through them.
With this feature one could "flatten" this structure and view all at the same time.

But it is important that this is toggleable, because I don't always want this on and it comes at a performance cost.

**Comment by sondhg** [comment]
Yes, something like `feh -r` or `feh --recursive` for `qimgv` would be a godsend. `qimgv` is the best image viewer I've ever used by far, but it missing recursive mode is the only reason I still keep `feh` installed.
